### ISSUE TITLE
All messages are treated as if they were the first

### ISSUE THREAD

**Issue by TiagoTiago** [oobabooga/text-generation-webui#1974]
### Describe the bug

I haven't figured out exactly what's triggering it, but sometimes when using the web UI to chat, only the current message is shown to the AI and the rest of the chat history is ignored.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

I haven't figured out what's triggering it, but once it starts happening, it's easy to confirm if you have the system in verbose mode, as you can see in the console that only the last message is shown to the AI.

### Logs

```shell
-
```

### System Info

```shell
Running with docker on Linux
```

**Comment by kuso-ge** [comment]
What are your parameter settings? IF you set the max-new-token to 2000(default maximum-unless you changed the max if you were using the MPT model), the AI will ignore context/chat history.

**Comment by TiagoTiago** [comment]
> What are your parameter settings? IF you set the max-new-token to 2000(default maximum-unless you changed the max if you were using the MPT model), the AI will ignore context/chat history.

Oh, that seems to be it. Why does new tokens affects context size? Is it done to avoid using more memory? Or is it actually a technical limitation even if I have enough memory available?

**Comment by Ph0rk0z** [comment]
Context size is input + output together. So the more tokens you want back, the less you can send to the model.

**Comment by Ph0rk0z** [comment]
Yes. all LLM so far. The context window is for both. Sometimes you want more tokens generated than you are sending as memory, especially in other modes.
